composer.json

```diff
@@ -15,7 +15,9 @@
         "rapidez/account": "~0.10",
         "rapidez/core": "~0.55",
         "rapidez/image-resizer": "~0.5",
-        "rapidez/menu": "~0.2"
+        "rapidez/menu": "~0.2",
+        "rapidez/reviews": "^0.5.2",
+        "rapidez/wishlist": "~0.3.0"
```

Comment:
These are currently optional: https://docs.rapidez.io/0.x/packages.html#optional, we like to give this installed by default to all new users? If so; please use `~` and don't specify the patch number.